### PR TITLE
Update learning_resources.md

### PR DESCRIPTION
**Pull request by sbaumohl**
OVER 9000 RESOURCES
just some stuff I've accumulated over the years.

### CODE REVIEW

File: introduction/learning_resources.md [hunk 1]
```diff
@@ -277,16 +278,26 @@ Courses | Duration | Effort
...
 #### Coursera: [Specializations](https://www.coursera.org/specializations)
 #### Udacity: [Nanodegree](https://www.udacity.com/nanodegree)
 #### FutureLearn: [Collections](https://www.futurelearn.com/courses/collections)
+#### Pluralsight: [Careers](https://www.pluralsight.com/browse/business-professional/careers)
 
-## More resource lists!
+## More resources and resource lists!
 Just like a good piece of software, this tutorial must be dependent on other lists. 
 
 - [Delevoper-Y videos](https://github.com/Developer-Y/cs-video-courses)
```
Comment (michaelgira23) [inline introduction/learning_resources.md]: I too am a software "Delevoper".

**Review by michaelgira23 — CHANGES_REQUESTED** [review]
Could you maybe fix this one thing before merge? Thanks!

File: introduction/learning_resources.md (hunk 1 above)
Comment (sbaumohl) [reply introduction/learning_resources.md]: Lol, okay, I'll fix it by tommorow night

**Comment by sbaumohl** [comment]
all good, I think.

**Review by michaelgira23 — APPROVED** [review]

**Pull request merged**